### DISCUSSION TITLE
T2I-Adapters for SD 1.5 released

### DISCUSSION THREAD

**Discussion by dfaker** [Mikubill/sd-webui-controlnet#609] [General]
https://huggingface.co/TencentARC/T2I-Adapter/tree/main/models

Still trying to bodge together the yamls but this looks promising.

**Comment by a3nima** [comment]
whats the difference to the ti2adapter models? they are trained on 1.5 instead of 1.4 ? can they be used the same way?

**Reply by Gochomer** [reply]
https://github.com/Mikubill/sd-webui-controlnet/issues/614

It seem to be an improved version of what this extension could do with several models working at the same time.

**Reply by ClashSAN** [reply]
the fuser adapter would have to be supported to enhance the experience when using multiple adapters, but the other 5 work exactly the same:

coadapter-canny, coadapter-sketch with sketch_adapter.yaml
coadapter-depth with image_adapter.yaml

**Reply by illtellyoulater** [reply]
@ClashSAN so we need to still use `sketch_adapter_v14.yaml` and `image_adapter_v14.yaml` but rename them as `xxxxxxxx_sd15v2.yaml`, correct? 
Like so?

```
- for t2iadapter_canny_sd15v2.pth : sketch_adapter_v14.yaml > t2iadapter_canny_sd15v2.yaml
- for t2iadapter_sketch_sd15v2.pth : sketch_adapter_v14.yaml > t2adapter_sketch_sd15v2.yaml
- for t2iadapter_depth_sd15v2.pth : image_adapter_v14.yaml > t2adapter_depth_sd15v2.yaml
```

**Comment by MegaScience** [comment]
So the `coadapter` ones, that don't have their own `keypose` and `openpose`? Interesting... They also changed the structure of the repository, so the download link for the preprocessors need to be updated, but that's a separate matter.

**Comment by amir84ferdos** [comment]
dose anybody knows how to use the Coadaopter Fuser?

**Comment by Milor123** [comment]
> https://huggingface.co/TencentARC/T2I-Adapter/tree/main/models
> 
> Still trying to bodge together the yamls but this looks promising.

where are the .yaml files ??

**Reply by nnqq2603** [reply]
https://github.com/Mikubill/sd-webui-controlnet/tree/main/models
